mne/bem.py

```diff
@@ -253,16 +253,19 @@ def _fwd_bem_ip_modify_solution(solution, ip_solution, ip_mult, n_tri):
     return
 
 
-def _check_complete_surface(surf, copy=False):
+def _check_complete_surface(surf, copy=False, incomplete='raise'):
     surf = complete_surface_info(surf, copy=copy, verbose=False)
     fewer = np.where([len(t) < 3 for t in surf['neighbor_tri']])[0]
     if len(fewer) > 0:
-        raise RuntimeError('Surface %s has topological defects: %d / %d '
-                           'vertices have fewer than three neighboring '
-                           'triangles [%s]'
-                           % (_surf_name[surf['id']],
-                              len(fewer), surf['ntri'],
-                              ', '.join(str(f) for f in fewer)))
+        msg = ('Surface %s has topological defects: %d / %d '
+               'vertices have fewer than three neighboring '
+               'triangles [%s]' % (_surf_name[surf['id']],
+                                   len(fewer), surf['ntri'],
+                                   ', '.join(str(f) for f in fewer)))
+        if incomplete == 'raise':
+            raise RuntimeError(msg)
+        else:
+            warn(msg)
```

Comment:
why is this here???

Reply:
got here likely by accident in 8d1023ee13da529940e50949eb06e9ee0e94594b